lib/membrane/rtp/serializer.ex

```diff
@@ -24,13 +25,37 @@ defmodule Membrane.RTP.Serializer do
                 """
               ]
 
+  defmodule State do
+    @moduledoc false
+    use Bunch.Access
+
+    defstruct sequence_number: 0,
+              init_timestamp: 0,
+              any_buffer_sent?: false,
+              stats_acc: %Stats{
```

Comment:
This is actually not `stats_acc` field, but actual `Stats`. Maybe let's call it 'stats' and init with `:no_stats`?

Reply:
okay